app/lib/package/name_tracker.dart

```diff
@@ -52,6 +93,15 @@ class NameTracker {
     _names.remove(name);
   }
 
+  /// Returns the cached list of packages ordered by descending last published date.
+  /// Only the visible packages are present.
+  List<TrackedPackage> get packagesOrderedByLastPublishedDesc {
```

Comment:
```suggestion
  List<TrackedPackage> get visiblePackagesOrderedByLastPublished {
```

I'm not sure its worth store the sorted list in-memory, but I don't mind it... it's one more risk of cache issues, but probably not going to happen.

Reply:
I think it is not a big overhead, and we don't need to sort the list every time it is accessed.